ramin/Data_structure/queue.js

```
@@ -0,0 +1,70 @@
+let myQueue = {
+
+    array : [],
+
+    isEmty : function (){
+        return this.array.length === 0;
+    },
+
+    insert : function(num){
+        this.array.push(num);
+    },
+
+    remove : function(){
+        if(this.isEmty())
+            console.log("Queue is empty");
```

its better to throw an error